adds some context around the review cadence for PRs in the PR template (#473)
Open-Cap-Table-Coalition/Open-Cap-Format-OCF: .github/PULL_REQUEST_TEMPLATE.md

1. Read the [contributing guidelines](../docs/CONTRIBUTING.md).
2. Ensure you have added or ran the appropriate tests for your PR.
3. If your PR is unfinished, consider creating a [Draft PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests).

> 4. PRs are discussed, reviewed, and responded to on a bi-weekly basis by the Technical Working Group. We'll get back to you soon!

Reviewer: ```suggestion
4. PRs are discussed, reviewed, and responded to every two weeks by the Technical Working Group. We'll get back to you soon!
```

Eliminates the ambiguity of bi-weekly and its cousin semi-weekly.
